src/NzbDrone.Core/Datastore/Migration/074_add_add_options_to_series.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(74)]
+    public class add_add_options_to_series : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Series").AddColumn("AddOptions").AsString().Nullable();
```

Comment:
no, It'll likely stay to avoid the need of chaining different commands, Can't ignore episodes until the series has been updated and AddOptions was part of the add command,

unless we add an overload to the UpdateSeries method.